createmobilespec/createmobilespec.js

```diff
@@ -498,6 +522,15 @@ function pluginIdToDirName(id) {
 ////////////////////// install plugins for each platform
 function installPlugins() {
     var plugins = DEFAULT_PLUGINS;
+
+    // special override for osx
+    if (argv.osx) {
+        if (platforms.length > 1) {
```

Comment:
This will override the plugins for all platforms, which is incorrect behaviour. Perhaps make the logic override plugins when OSX is the *only* specified platform?